renovate.json

```diff
@@ -1,6 +1,39 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "@atlassian:fusion-arc-renovate-config"
+    "config:base"
+  ],
+  "reviewers": [
+    "bgvozdev",
+    "atrigueiro",
+    "rrathbone",
+    "mboudreau",
+    "jkay",
+    "gxue",
```

Comment:
Hi @rachellerathbone can you update this to `gxueatlassian`, I think that's my username.